server/selection_test.go

```diff
@@ -1,379 +0,0 @@
-package server
```

Comment:
@leszko why were these tests removed. Although the tests done in here are quite trivial they still are related to the main logic in the `selection.go` file and not the new `selection_algorithm.go`.

Reply:
There should not be removed. I'll take them back after merging this one: https://github.com/livepeer/go-livepeer/pull/3188

Otherwise, they fail 🙃

Reply:
No problem thanks!